cloud-controller-manager/do/firewall_controller.go

```diff
@@ -312,14 +313,39 @@ func (fm *firewallManager) createReconciledFirewallRequest(serviceList []*v1.Ser
 					},
 				)
 			}
+		} else if svc.Spec.Type == v1.ServiceTypeLoadBalancer {
+			lbType, err := getType(svc)
+			if err != nil {
+				return nil, err
```

Comment:
Can we add `fmt.Errorf()` around `err` to provide context, specifically the name of the Service affected?

Similar for the error returned in line 323.